Converters/Tekla/Speckle.Converter.Tekla2024/ToSpeckle/Helpers/DisplayValueExtractor.cs

```diff
@@ -0,0 +1,48 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Sdk.Models;
+using SOG = Speckle.Objects.Geometry;
+
+namespace Speckle.Converter.Tekla2024.ToSpeckle.Helpers;
+
+public sealed class DisplayValueExtractor
+{
+  private readonly ITypedConverter<TSM.Solid, SOG.Mesh> _meshConverter;
+  private readonly IConverterSettingsStore<TeklaConversionSettings> _settingsStore;
+
+  public DisplayValueExtractor(
+    ITypedConverter<TSM.Solid, SOG.Mesh> meshConverter,
+    IConverterSettingsStore<TeklaConversionSettings> settingsStore
+  )
+  {
+    _meshConverter = meshConverter;
+    _settingsStore = settingsStore;
+  }
+
+  public IEnumerable<Base> GetDisplayValue(TSM.ModelObject modelObject)
+  {
+    switch (modelObject)
+    {
+      case TSM.Beam beam:
+        var solid = beam.GetSolid();
```

Comment:
Is `GetSolid()` on the `Part` level? If so we can just have one `TSM.Part` case to retrieve the solid from